src/main/java/com/solr/clientwrapper/domain/service/DataIngectionService.java

```diff
@@ -0,0 +1,64 @@
+package com.solr.clientwrapper.domain.service;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.solr.clientwrapper.domain.port.api.DataIngectionServicePort;
+
+@Service
+@Transactional
+public class DataIngectionService  implements DataIngectionServicePort{
+	
+	private final Logger log = LoggerFactory.getLogger(DataIngectionService.class);
+	SolrClient solrClient=new HttpSolrClient.Builder("http://localhost:8983/solr/docparse").build();
+	
+	@Override
+	public String parseSolrSchemaArray(String jsonString) {
+
+		JSONObject jsonObject = new JSONObject(jsonString);
+		log.debug("json Object :-{}", jsonObject);
+		JSONArray jArray = new JSONArray();
+
+		for (@SuppressWarnings("rawtypes")
+		Iterator iterator = jsonObject.keySet().iterator(); iterator.hasNext();) {
+			String key = (String) iterator.next();
+			jArray = (JSONArray) jsonObject.get(key);
+
+		}
+		log.debug("input array :-{}", jArray.toString());
+		return jArray.toString();
+	}
+
+	@Override
+	public String parseSolrSchemaBtch(String jsonString) {
```

Comment:
Please comment down below, how the structure of jsonString will be

Reply:
Ravikiran's code. The whole DataInjection layer.

Reply:
The expected structure is as follows by the way(as shared and told by Ravikiran):
[JsonStrct.txt](https://github.com/NeoSOFT-Technologies/search-as-service/files/7769325/JsonStrct.txt)